web-app/js/portal/data/LayerStore.js

```diff
@@ -31,41 +31,15 @@ Portal.data.LayerStore = Ext.extend(GeoExt.data.LayerStore, {
 
     addUsingLayerLink: function(layerDisplayName, layerLink, geonetworkRecord, layerRecordCallback) {
         var serverUri = layerLink.server.uri;
+        var serverInfo = Portal.data.Server.getInfo(serverUri);
 
-        Ext.Ajax.request({
-            url: 'server/getInfo?' + Ext.urlEncode({server: serverUri}),
-            scope: this,
-            success: function(resp) {
-                try {
-                    var serverInfo = Ext.util.JSON.decode(resp.responseText);
-                    this._serverInfoLoaded(layerDisplayName, layerLink, geonetworkRecord, layerRecordCallback, serverInfo);
-                }
-                catch (e) {
-                    log.error("Failed parsing information for server '" + serverUri + "', response : '" + resp.responseText + "'");
-                }
-            },
-            failure: function(resp) {
-                log.error("Failed getting information for server '" + serverUri + "'");
-                this._serverUnrecognized(layerDisplayName, layerLink);
-            }
-        });
-    },
-
-    _serverInfoLoaded: function(layerDisplayName, layerLink, geonetworkRecord, layerRecordCallback, serverInfo) {
-        if ($.isEmptyObject(serverInfo)) {
-
+        if (!serverInfo) {
             this._serverUnrecognized(layerDisplayName, layerLink);
-
-            // break this layer as the server is unrecognised
-            // openLayers needs the minimum to attempt loading #1476
-            layerLink.server = {
-                uri: this.BLOCKED,
-                type: this.BLOCKED
-            };
-            layerRecordCallback = undefined;
-            geonetworkRecord = undefined;
+            return;
```

Comment:
`return`ing here changes the behaviour from before; previously it went on to call the _addUsingLayerLinkNcwms or _addUsingLayerLinkDefault methods. If this is deliberate then can we do away with the fake server we assign to `layerLink.server`?